examples/bri-3/src/bri/zeroKnowledgeProof/services/blockchain/ethereum/contracts/Ccsm.sol

```diff
@@ -0,0 +1,16 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
+contract Ccsm {
+  mapping(string => bool) public anchorHashStore;
+
+  function setAnchorHash(string calldata _anchorHash) external {
+    anchorHashStore[_anchorHash] = true;
+  }
+
+  function getAnchorHash(
```

Comment:
Yeah anchorHashStore is publicly available. I added this function in case we wanted to add any verification with the getFunction. If not required, I'll take it out in a future PR.